src/app/files/fileManager.js

```diff
@@ -98,7 +99,7 @@ class FileManager {
     $filesEl.find('.file').remove()
 
     for (var file in this.tabbedFiles) {
-      $filesEl.append($('<li class="file"><span class="name">' + file + '</span><span class="remove"><i class="fa fa-close"></i></span></li>'))
```

Comment:
Oh wow, so I guess this is place of the bug, isn't it?

Reply:
yes it is